### PR TITLE
update rbac api version to v1

### PR DESCRIPTION
**Pull request by lawrencedudley**
The v1alpha1 isn't used by current versions of Kubernetes - this should be updated in the example docs

### CODE REVIEW

**Comment by lawrencedudley** [comment]
Signed!

**Comment by lawrencedudley** [comment]
One more try...

**Comment by rootfs** [comment]
@lawrencedudley thanks can you update the rest too?

**Comment by lawrencedudley** [comment]
@rootfs that's done!

File: vendor/k8s.io/client-go/CHANGELOG.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Bug fix: picked up a security fix [kubernetes/kubernetes#53443](https://github.c
 
    * [https://github.com/kubernetes/kubernetes/pull/41901](https://github.com/kubernetes/kubernetes/pull/41901)
 
-* Promoted rbac.authorization.k8s.io/v1beta1 to rbac.authorization.k8s.io/v1
```
Comment (rootfs) [inline vendor/k8s.io/client-go/CHANGELOG.md]: don't change vendor code

**Comment by rootfs** [comment]
thanks @lawrencedudley, one more thing then lgtm

**Comment by lawrencedudley** [comment]
Sorry @rootfs that's me being an idiot. Should be good now!

**Comment by lawrencedudley** [comment]
retest this please

**Comment by rootfs** [comment]
CI failed on yum install, not related to this fix

/lgtm

cc @childsb for merge

**Pull request merged**